modules/swagger-codegen/src/main/resources/objc/api-header.mustache

```diff
@@ -21,8 +21,8 @@ extern NSInteger k{{classname}}MissingParamErrorCode;
 /// {{#allParams}}@param {{paramName}} {{description}}{{^required}} (optional){{/required}}{{#defaultValue}} (default to {{{.}}}){{/defaultValue}}
 /// {{/allParams}}{{#responses}}
 ///  code:{{{code}}} message:"{{{message}}}"{{#hasMore}},{{/hasMore}}{{/responses}}
-///
-/// @return {{{returnType}}}
+///{{#returnType}}
+/// @return {{{returnType}}}{{/returnType}}
```

Comment:
@gregra81 I've submitted https://github.com/swagger-api/swagger-codegen/pull/4437 for the enhancement.

If you find other warnings with the ObjC client, please open a new "issue" for tracking.